utils/db_train_test_split_by_user.py

```diff
@@ -1,11 +1,15 @@
 from random import shuffle
 import pandas as pd
 import pymongo
+from sklearn.model_selection import train_test_split
+import re
 
 config = {
     "DB_HOST": "mongodb://host.docker.internal:27017/",
-    "DB_NAME": "behaworks_logger_v6",
-    "DB_NAME_OTHER": "behaworks_logger_v6_napodobnovanie_split_by_name"
+    "DB_NAME": "behaworks_logger_v8",
+    "DB_NAME_OTHER": "behaworks_logger_v8",
+    "RATIO_OF_EXISTING_USERS": 70,
+    "NORMAL_SPLIT": "y"
```

Comment:
Nemas pre toto nejaky lepsi nazov? 
Nemoze to byt radsej bool True/False?

Reply:
nazov ma lepsi nenapadol, rozhoduje to o tom ci sa budu data delit novym sposobom (v teste aj novy aj existujuci) alebo na napodobnovany - napodobnujuci. ak mas nejaky nazov ktory to vystihuje tak to kludne zmenim 

True/False tam kludne mozem dat